### DISCUSSION TITLE
Is there a general stats page (not podcast specific)?

### DISCUSSION THREAD

**Discussion by dellagustin** [skymethod/op3#29] [Q&A]
Hello @johnspurlock-skymethod ,

Does op3 have a general stats page that show aggregated information for all podcasts using it?
At the moment I'm mainly looking for information on the "market share" (for lack of a better term) of podcast players.
The intent here is to prioritize open source contribution of Podcasting 2.0 features to the players that are more used, to get the best "return on investment" and optimize the speed of Podcasting 2.0 feature adoption.

Best Regards,
Guilherme.

**Comment by johnspurlock-skymethod** [comment]
A global stats page is number two on my current roadmap, hopefully this month.

Number one is a backend refactoring to bring [costs](https://op3.dev/costs) down in line with [sponsor](https://github.com/skymethod/op3?tab=readme-ov-file#sponsors) levels, and to allow scaling up to 10x the number of shows on OP3.

Consider getting your company to sponsor OP3 at the 100/mo level!

**Reply by Marzal** [reply]
Hi, I'm trying to familiarize me with the op3 web code (sysadmin here, no dev), in order to try to do a few accessibility MR.

Is [this file](https://github.com/skymethod/op3/blob/master/worker/static/stats.htm) a work in progress general stats page?

PD: And what most important, it [this one](https://github.com/skymethod/op3/blob/master/worker/static/show.htm) the right file to change each show stats page?

**Reply by johnspurlock-skymethod** [reply]
> Is [this file](https://github.com/skymethod/op3/blob/master/worker/static/stats.htm) a work in progress general stats page?

Yes, you can ignore that for now.  Will have "global" stats eventually, but I'd rather spend limited dev time on new podcaster-facing features first.

> And what most important, it [this one](https://github.com/skymethod/op3/blob/master/worker/static/show.htm) the right file to change each show stats page?

You got it - that's the show stats page.  It would probably help to add a page to the repo about how to get a local dev running for frontend-only changes like this.  I will take this as a todo.

**Comment by johnspurlock-skymethod** [comment]
Oh and in the meantime, you can get a [get the top apps across all OP3 shows](https://op3.dev/api/docs#tag/queries/operation/queryTopApps) via the OP3 API

Another global view of app share is to check out [Libsyn's statistics](https://livewire.io/libsyn-stats-visualized/) (more downloads) and [Buzzsprout's stats](https://livewire.io/buzzsprout-stats-visualized/) (fewer downloads).

Many ways to look at the elephant

**Reply by jamescridland** [reply]
I didn't know about this call! Excellent. *rubs hands*

**Reply by jamescridland** [reply]
https://podcastbusinessjournal.com/data/podcast-apps/ looks, well, functional.
